### PR TITLE
Add component sticky navbar and temporary bg img

### PR DESCRIPTION
**Pull request by Alteo96**

### CODE REVIEW

**Comment by Alteo96** [comment]
Also, we should discuss with Dragos how to make the page responsive.

File: client/src/components/navigation/Navigation.jsx [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+          <button className="italic ... hover:text-yellow-300 delay-150">Delivery</button>
+          <button className="italic ... hover:text-yellow-300 delay-150">Contacts</button>
+          <button className="italic ... hover:text-yellow-300 delay-150">Login</button>
+          <button className="italic ... hover:text-yellow-300 delay-150">Cart</button>
+        </nav>
+    </>
+  );
+}
+
+
```
Comment (strdr4605) [inline client/src/components/navigation/Navigation.jsx]: Maybe remove 2 extra blank lines.
Each file should have only one empty line at the end.

File: client/src/components/navigation/Navigation.jsx [hunk 2]
```diff
@@ -1,5 +1,17 @@
-export default function Navigation(){
-    return (
-        <nav>Navigation</nav>
-    )
-}
\ No newline at end of file
+export default function Navigation() {
+  return (
+    <>
```
Comment (strdr4605) [inline client/src/components/navigation/Navigation.jsx]: Looks like React fragment is redundant here, as it only has one child `<nav>`, maybe remove?😊

File: client/src/components/navigation/Navigation.jsx
Comment (strdr4605) [inline client/src/components/navigation/Navigation.jsx:1]: Looks to me that we have inconsistent styles of writing JS in this project.
|Before|After|
|------|-----|
|ndentation with 4 spaces| 2 spaces|
|`Navigation(){`|`Navigation() {`|
| no semicolons| semicolons |

I would suggest adding eslint and prettier (+ VSCode plugins) to this project for consistency across the team.

File: client/src/App.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+body {
+    background-image: url('https://images.unsplash.com/photo-1604382354936-07c5d9983bd3?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80');
+}
```
Comment (strdr4605) [inline client/src/App.css]: Add empty line at end of file😊

File: client/src/App.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (strdr4605) [inline client/src/App.css]: Looks to me that changes for `body` should be in https://github.com/ghilannicoleta/la-fontana-app/blob/main/client/src/index.css#L5.
And `App.css` should be removed from the project, as this project uses tailwind

**Review by strdr4605 — CHANGES_REQUESTED** [review]
Would be nice to have a screenshot of the UI changes, so there is no need to checkout the branch locally on every small UI change.

> Hint: You can copy-paste the screenshot in the PR description. Sometimes "before" and "after" screenshot is also useful.
On macOS: CMD+Shift+4 to capture a section of the screen
----
About the responsiveness. I am not familiar with tailwindcss but probably this should do the job: https://tailwindcss.com/docs/responsive-design

**Review by MaryPirtu — APPROVED** [review]

**Pull request merged**